src/website/app/demos/Button/bestPractices.js

```diff
@@ -21,38 +21,35 @@ import Button from '../../../../Button';
 export default [
   {
     type: 'do',
-    title: 'use the appropriate variant for your intent',
-    description: `Match Mineral UI's pre-defined Button variants with semantic purpose.`,
+    description:
+      'Use the [appropriate variant](/color/#guidelines-variants) for your intent.',
     example: (
-      <Button variant="danger" primary>
-        Drop Users Table
+      <Button variant="success" primary>
+        Proceed to Checkout
       </Button>
     )
   },
-  {
-    type: 'do',
-    title: 'use clear labeling for button messaging',
-    description: `Button actions should be predictable for frictionless interaction.
-Exceptions would include "Ok", "Cancel", etc. which should be used sparingly.
-Labels should be structured: \`<verb> <noun>\`.`,
-    example: <Button primary>Save Changes</Button>
-  },
   {
     type: 'dont',
-    title: 'cause confusion with the wrong variant',
     description:
-      'Using the wrong variant can cause confusion, or undue stress for users.',
+      "Avoid using a variant that doesn't match intent, which will confusion.",
```

Comment:
I read this as, "Don't avoid using a variant that doesn't match intent..." which is hard to read because, besides being a triple-negative, "Don't" is not the first word of the phrase. I think in these examples, we should have the phrases start with the word, "Don't" followed by the action we shouldn't do, and provide the reason in a second sentence.

Reply:
That's a good suggestion. I was trying to keep the content agnostic of the Do/Don't titles, and even tried to avoid starting with "Don't" whenever possible. But that invites the misreading as you point out.

Just pushed a commit to address these, if you want to review again.